backend/src/config/auth.py

```
@@ -1,10 +1,93 @@
+    SELECT user_id, source, credentials, is_admin
+    FROM credentials
+    WHERE 
+        expires_at > NOW()
+        AND needs_refresh_at < NOW()
+    LATEST ON issued_at PARTITION BY user_id, source
+    """
+
+    return execute_query(pool, query)
+
+
+def user_is_admin(logto_token: str):
```

Ruff (0.14.14)

[warning] Unused function argument: `logto_token`

(ARG001)